tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -2118,6 +2153,54 @@ def func(self, xp, a):
         return xp.arctan(a)
 
 
+# Since the gradient of arctan2 is quite flaky.
+# for smaller values especially `float16`.
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Special shapes
+    chainer.testing.product({
+        'in_shapes': _shapes_combination_binary,
+        'in_dtypes,out_dtype': (
+            _make_same_in_out_dtypes(2, chainerx.testing.float_dtypes)),
+        'input_lhs': ['random'],
+        'input_rhs': ['random'],
```

Comment:
Could you make these values deterministic to avoid stochastic test failure?

Reply:
Ah sure.

Reply:
Fixed. Thank You.